src/index.html

```diff
@@ -9,6 +9,22 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Search bar airbnb</h1>
+  <form data-qa="big" class="form-for-searching">
+  <input
+    class="search_bar"
+    type="search"
+    name="big_search_bar"
+    placeholder="Try &#34Los Angeles&#34"
+    data-qa="keypress"
+    >
+  </form>
+  <form data-qa="small" class="form-for-searching">
+  <input
+  class="search_bar search_bar--small"
+  type="search"
+  name="small"
+  placeholder="Try &#34Los Angeles&#34"
+  >
```

Comment:
Follow the indentation rules. Add an additional space of 2 spaces for nested tags/text.

Reply:
```
 <input
  class="search_bar search_bar--small"
  type="search"
  name="small"
  placeholder="Try &#34Los Angeles&#34"
>
```

Reply:
```
 <input
  class="search_bar search_bar--small"
  type="search"
  name="small"
  placeholder="Try &#34Los Angeles&#34"
>
```